10:01:40 [TGFSystem-akka.actor.default-dispatcher-29] WARN  akka.remote.artery.ArteryTransport - Other node [akka://ChargingSystem@127.0.0.1:25521#-337230311009628481] quarantined this node.
10:01:39 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:40 [TGFSystem-akka.actor.default-dispatcher-29] WARN  akka.remote.artery.Association - Association to [akka://ChargingSystem@127.0.0.1:25521] with UID [-337230311009628481] is irrecoverably failed. UID is now quarantined and all messages to this UID will be delivered to dead letters. Remote ActorSystem must be restarted to recover from this situation. Reason: Idle longer than quarantine-idle-outbound-after [6.000 h]
10:01:40 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:40 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1102 ms, attempt: 3, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:41 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:41 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:41 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1157 ms, attempt: 4, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:42 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:42 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:42 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1214 ms, attempt: 5, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:44 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:44 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:44 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1274 ms, attempt: 6, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:45 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:45 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:45 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1337 ms, attempt: 7, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:46 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:46 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:46 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1403 ms, attempt: 8, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:48 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:48 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:48 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1473 ms, attempt: 9, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:49 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:49 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:49 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1546 ms, attempt: 10, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:51 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:51 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:51 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1623 ms, attempt: 11, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:52 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /172.17.0.2:5701
10:01:52 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to [51.120.121.91]:5701: com.hazelcast.core.HazelcastException: java.io.IOException: Network is unreachable: getsockopt to address /51.120.121.91:5701
10:01:52 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Unable to get live cluster connection, retry in 1704 ms, attempt: 12, cluster connect timeout: INFINITE, max backoff: 30000 ms
10:01:59 [hz.client_1.internal-9] WARN  com.hazelcast.client.impl.connection.ClientConnectionManager - hz.client_1 [dev] [5.4.0] Exception during initial connection to Member [172.17.0.2]:5701 - 8e792194-15f8-4127-9f87-d56a68ff4a3d: com.hazelcast.core.HazelcastException: java.io.IOException: Connect timed out to address /172.17.0.2:5701
10:24:20 [TGFSystem-akka.actor.default-dispatcher-5] WARN  akka.remote.RemoteActorRefProvider - Using the 'remote' ActorRefProvider directly, which is a low-level layer. For most use cases, the 'cluster' abstraction on top of remoting is more suitable instead.
10:24:20 [TGFSystem-akka.actor.default-dispatcher-5] WARN  akka.remote.RemoteActorRefProvider - Akka Cluster not in use - Using Akka Cluster is recommended if you need remote watch and deploy.
10:32:23 [TGFSystem-akka.actor.default-dispatcher-22] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], control stream] Upstream failed, cause: StreamTcpException: The connection has been aborted
10:32:23 [TGFSystem-akka.actor.default-dispatcher-22] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: The connection has been aborted
10:58:19 [TGFSystem-akka.actor.default-dispatcher-4] WARN  akka.remote.RemoteActorRefProvider - Using the 'remote' ActorRefProvider directly, which is a low-level layer. For most use cases, the 'cluster' abstraction on top of remoting is more suitable instead.
10:58:19 [TGFSystem-akka.actor.default-dispatcher-4] WARN  akka.remote.RemoteActorRefProvider - Akka Cluster not in use - Using Akka Cluster is recommended if you need remote watch and deploy.
11:15:37 [TGFSystem-akka.actor.default-dispatcher-29] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:15:46 [TGFSystem-akka.actor.default-dispatcher-28] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:15:55 [TGFSystem-akka.actor.default-dispatcher-28] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:16:04 [TGFSystem-akka.actor.default-dispatcher-24] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:16:13 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:16:22 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:16:31 [TGFSystem-akka.actor.default-dispatcher-28] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:16:40 [TGFSystem-akka.actor.default-dispatcher-28] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:16:49 [TGFSystem-akka.actor.default-dispatcher-28] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:16:58 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:17:07 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:17:16 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:17:25 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:17:34 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:17:43 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:17:52 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:18:01 [TGFSystem-akka.actor.default-dispatcher-28] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:18:10 [TGFSystem-akka.actor.default-dispatcher-33] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
11:18:19 [TGFSystem-akka.actor.default-dispatcher-28] WARN  akka.stream.Materializer - [outbound connection to [akka://ChargingSystem@127.0.0.1:25521], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(127.0.0.1/<unresolved>:25521,None,List(),Some(5000 milliseconds),true)] failed because of java.net.ConnectException: Connection refused: getsockopt
